tests/backup/backup_test.go

```diff
@@ -47,9 +60,15 @@ func TestBackup(t *testing.T) {
 var _ = BeforeSuite(func() {
 	InitInstance()
 	CreateOrganization(orgID)
```

Comment:
@stgleb CreateOrganization() call itself fail for other non-backup jenkins jobs (jobs that are not configured to run backup tests)

Reply:
well, I can put  a code for determining cloud provider by creds that will go  out of this BeforeSuite if this job was not run for backup.